### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_ReactLogin_DevTools: The role is not defined for "Remember me" checkbox on the login page.

### ISSUE THREAD

**Issue by Truptik12** [dotnet/aspnetcore#42342]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG4.1.2;#DesktopApp;#Win11;#Benchmark;#Devtools;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2

### Steps to Reproduce:
1. Open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir React"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new react -au Individual" and hit enter.
5. Run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. Browser page will open with above template loaded
9. Tab to the Login page.
10. Tab to the "Remember me" checkbox and inspect it.

### Actual:
The role is not defined for the "Remember me" checkbox on the login page.

This issue is observed throughout the application for all the Login pages.

### Expected:
The role should be defined for "Remember me" checkbox on the login page.

### User Impact:
Screen reader users will not be able to get the correct role type for the selected control.

**Comment by Truptik12** [comment]
Closing the bug as the issue is not reproducing anymore.

**Comment by mkArtakMSFT** [comment]
Issue is fixed. Adding #Closed; tag.
